code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -89,7 +89,7 @@
 
 /obj/item/gun/projectile/pistol/throwback
 	name = "pistol"
-	desc = "A product of one of thousands of illegal workshops from around the galaxy. Often replicas of ancient Earth handguns, these guns are usually found in hands of frontier colonists and pirates."
+	desc = "A product of one of thousands of illegal workshops from around the world. Often replicas of ancient handguns, these guns are usually found in hands of pirates and low-lifes."
```

Comment:
Uhhhhhhh, handguns aren't "ancient" and as far as I know we don't have pirates in the modern day.

Reply:
yes we do have pirates in the modern day and this needs better wording, it's a draft.